lark/load_grammar.py

```diff
@@ -231,12 +231,11 @@ def will_not_get_removed(sym):
                 return not sym.name.startswith('_')
             if isinstance(sym, Terminal):
                 return keep_all_tokens or not sym.filter_out
-            assert False
+            if isinstance(sym, Symbol):
```

Comment:
This should not be needed. There are only two Symbols subclasses: `NonTerminal` and `Terminal`.

Reply:
Right now `_EMPTY` is a Symbol, and isn't a terminal or nonterminal.

But it probably makes more sense to test `is _EMPTY`.

Reply:
yea testing for ``is _EMPTY`` would make more sense. i went with ``isinstance(sym, Symbol)`` because ``Symbol`` is only ever called directly when defining ``_EMPTY``, but it's less clear and less precise than testing for ``is _EMPTY``.